src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/SymbolModel/ComputedSymbol.cs

```diff
@@ -2,9 +2,9 @@
 using Microsoft.TemplateEngine.Utils;
 using Newtonsoft.Json.Linq;
 
-namespace Microsoft.TemplateEngine.Orchestrator.RunnableProjects
+namespace Microsoft.TemplateEngine.Orchestrator.RunnableProjects.SymbolModel
 {
-    public class ComputedSymbol : ISymbolModel
+    public class ComputedSymbol : ISymbolModel, ISymbolModel2
```

Comment:
Do we need an `ISymbolModel2`? If anyone who implements `ISymbolModel` is also going to implement `ISymbolModel2`, can't we just use `ISymbolModel` and add the `FileRename` property into it?

Reply:
This was original implementation, however after discussion with @AR-May I understood that modifications to existing public interfaces are not desirable.